### PR TITLE
Add pep257 package

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @goanpeca. Also, could you please rebase as there is some clutter that just cleared?

**Comment by goanpeca** [comment]
Sure

File: recipes/pep257/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    can be easily added, for example to cover NumPy docstring conventions.
+    pep257 supports Python 2.6, 2.7, 3.2, 3.3, 3.4, pypy and pypy3.
+  doc_url: http://pep257.readthedocs.io/en/latest
+  dev_url: http://github.com/GreenSteam/pep257
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - Nurdok
+
```
Comment (jakirkham) [inline recipes/pep257/meta.yaml]: nit: please remove extra newline. One terminal newline is sufficient.

File: recipes/pep257/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    The framework for checking docstring style is flexible, and custom checks
+    can be easily added, for example to cover NumPy docstring conventions.
+    pep257 supports Python 2.6, 2.7, 3.2, 3.3, 3.4, pypy and pypy3.
+  doc_url: http://pep257.readthedocs.io/en/latest
+  dev_url: http://github.com/GreenSteam/pep257
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - Nurdok
```
Comment (jakirkham) [inline recipes/pep257/meta.yaml]: @Nurdok, is it alright if include you on `pep257`?

We are aware it is being phased out and replaced by `pydocstyle`, but some packages have yet to update the dependency. So, we would like to keep it around ATM for that reason. As a result, we expect the effort spent here to be negligible.

**Comment by jakirkham** [comment]
One nit and question for the added maintainer. Otherwise LGTM. Thanks @goanpeca.

**Comment by goanpeca** [comment]
:+1:

I assumed @Nurdok was ok with based on his answer with pydocstyle

**Comment by goanpeca** [comment]
I will remove him for now so we can merge this @jakirkham, ok?

File: recipes/pep257/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pep257
+
+about:
+  home: https://github.com/GreenSteam/pep257
+  license: MIT License
```
Comment (jakirkham) [inline recipes/pep257/meta.yaml]: Oops, missed this. Let's shorten this to "MIT".

Comment (goanpeca) [inline recipes/pep257/meta.yaml]: :+1:

**Comment by jakirkham** [comment]
Missed one point. Once addressed happy to merge.

**Comment by goanpeca** [comment]
Good to go

**Comment by jakirkham** [comment]
Thanks @goanpeca.
